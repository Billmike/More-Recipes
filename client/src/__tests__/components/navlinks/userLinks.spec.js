import React from 'react';
import { shallow } from 'enzyme';
import UserLinks from '../../../components/navlinks/userLinks';

describe('<UserLinks />', () => {
  it('Should render the UserLinks component correctly', () => {
    const wrapper = shallow(<UserLinks />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('ul').length).toBe(1);
    expect(wrapper.find('li').length).toBe(6);
  });
});
