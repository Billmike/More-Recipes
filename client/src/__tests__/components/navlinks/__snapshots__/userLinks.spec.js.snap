// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserLinks /> Should render the UserLinks component correctly 1`] = `
<ul
  className="nav navbar-nav ml-auto"
>
  <li
    className="nav-item active"
  >
    <Link
      className="nav-link custom-link"
      id="home-button"
      replace={false}
      to="/"
    >
      Home
      <span
        className="sr-only"
      >
        (current)
      </span>
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link custom-link"
      replace={false}
      to="/dashboard"
    >
      My Recipes
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link custom-link"
      id="addRecipeID"
      replace={false}
      to="/add"
    >
      Add Recipe
    </Link>
  </li>
  <li>
    <Link
      className="nav-link custom-link"
      id="favoritesButton"
      replace={false}
      to="/favorites"
    >
      My Favorites
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link custom-link"
      replace={false}
      to="/profile"
    >
      My Profile
    </Link>
  </li>
  <li
    className="nav-item"
  >
    <Link
      className="nav-link logout btn-primary"
      id="logoutButton"
      replace={false}
      to="/"
    >
      Logout
    </Link>
  </li>
</ul>
`;
